src/main/java/cc/polyfrost/oneconfig/config/Config.java

```diff
@@ -270,6 +283,16 @@ protected final void generateOptionList(Object instance, Class<?> targetClass, O
                 optionNames.put(optionName, option);
             }
         }
+        if (!subMods.isEmpty()) {
```

Comment:
this is a rather hacky way to get this to work. i mean this is v0 so i guess its fine..